lib/spree/auth/engine.rb

```diff
@@ -60,10 +60,6 @@ def self.frontend_available?
         @@frontend_available ||= Gem::Specification.find_all_by_name('spree_frontend').any?
       end
 
-      def self.api_available?
```

Comment:
I'm not sure if this is safe to be removed, it's used e.g. in lib/spree/auth/engine.rb:43, to determine whether to load decorators for API classes.